tools/tasks/seed/build.html_css.ts

```diff
@@ -6,6 +6,7 @@ import * as cssnano from 'cssnano';
 import {join} from 'path';
 import {APP_SRC, TMP_DIR, CSS_PROD_BUNDLE, CSS_DEST, APP_DEST, BROWSER_LIST, ENV, DEPENDENCIES} from '../../config';
 const plugins = <any>gulpLoadPlugins();
+let cleanCss = require('gulp-clean-css');
```

Comment:
Should be and probably a good idea to use a TS standard `import` as opposed to requiring for the sake of consistancy